CCS2.0/DataLibrary/BusinessLogic/PhotoProccesor.cs

```diff
@@ -130,12 +130,27 @@ from dbo.Tag
 
         }
 
-        public static List<CommentModel> GetCommentId(int ImageId)
+        public static int GetPages(int ImageId)
         {
             var parameters = new { ImageId = ImageId };
+
+            string sql = @"SELECT count(*)
+                        FROM dbo.Comment
+                        Where ImageId = @ImageId;";
+
+            var page = SqlDataAccess.count<int>(sql, parameters)[0];
+            return page;
+        }
+
+        public static List<CommentModel> GetCommentId(int ImageId, int PageNumber)
+        {
+            var parameters = new { ImageId = ImageId,  PageNumber = PageNumber, RowOfPage = 5 };
```

Comment:
Ahh a duplicate of that 5 here... I'd centralize that either as a param or a setting.